vs

```diff
@@ -1,15 +1,19 @@
 #!/bin/bash
+# This is a small wrapper that lets VegaStrike - Upon the Coldest Sea
+# run from seemingly anywhere
 set -e
 
 VEGASTRIKE_LOCAL_SHARE_DIR="/usr/local/share/vegastrike"
 VEGASTRIKE_SHARE_DIR="/usr/share/vegastrike"
 
-# This is a small wrapper that lets VegaStrike - Upon the Coldest Sea
-# run from seemingly anywhere
+#Let's keep a log of the console
```

Comment:
off?

Reply:
The tee command writes it's input to the console & to the file specified. 
I don't want to fill the .vegastrike directory with files for people that are not interested, so my log rotation is using 2 files only.